secret/driver/plain.go

```diff
@@ -23,6 +24,10 @@ func (p *Plain) Encrypt(input string) (string, error) {
 	return input, nil
 }
 
-func (p *Plain) Type() string {
+func (p Plain) Type() string {
 	return "plain"
 }
+
+func (p Plain) GetKey() string {
+	return "plainTextDoesntHaveAKey"
```

Comment:
```suggestion
	return "plainDoesNotHaveAKey"
```
Same wording as base64 (no abbreviation)

Reply:
fixed.